core/src/test/java/org/apache/accumulo/core/file/rfile/MultiLevelIndexTest.java

```diff
@@ -18,6 +18,7 @@
  */
 package org.apache.accumulo.core.file.rfile;
 
+import static org.apache.accumulo.core.crypto.CryptoServiceFactory.newInstance;
```

Comment:
I like static imports for some things, but `newInstance` seems too generic and prone to error, since it looks like `Class.newInstance` and it isn't obvious it's using the CryptoServiceFactory.